OrthoEvol/Orthologs/Blast/comparative_genetics.py

```diff
@@ -73,34 +78,62 @@ def __init__(self, project=None, project_path=os.getcwd(), acc_file=None,
         self.__pre_blast = pre_blast
         self.__post_blast = post_blast
         self.__taxon_filename = taxon_file
+        self.acc_file = acc_file
         self.acc_csv_filename = acc_file
+
+        # Initialize variables
+        self.project_path = project_path
         self.project = project
+        self.species = 'Homo_sapiens'
+        self.taxon_file = taxon_file
```

Comment:
Same here.  `self.__taxon_filename` and `self.taxon_file` are given the same parameter value.

Reply:
Good catch, I wasn't aware of that.